.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,4 @@
-
-This PR addresses GitHub issue: # .
+GitHub issue resolved: #<number>. 
```

Comment:
I think it should be possible to add comment html style to these templates where we can explain that this line will automatically close the issue when merged. Those comments will only show up in edit mode and not in viewing mode.

Reply:
Great idea, done!